cleanup/README.md

```diff
@@ -0,0 +1,26 @@
+# Cleanup
+
+<p align="center">
+    <img src="logo.svg" width="400">
+</p>
+
+Cleanup is a simple service for cleaning server from old docker containers, images and volumes by cron schedule. 
+
+<table>
+  <tr>
+    <td align="center" colspan="2"><b>Cleanup</b></td>
+  </tr>
+
+  <tr>
+    <td>Volume</td>
+    <td><code>/var/run/docker.sock:/var/run/docker.sock</code></td>
+  </tr>
+  <tr>
+    <td valign="top">Base image</td>
+    <td><code>alpine:3.12</code></td>
+  </tr>
+</table>
+
+## Installation
+
+Read the [Getting Started](https://github.com/tolstoyevsky/mmb#getting-started) section to learn how to install this or other services.
```

Comment:
Add `Configuration` block and describe the environment variables in it. Like [here](https://github.com/tolstoyevsky/mmb/tree/add-cleanup/mediawiki#configuration).